posthog/models/web_preaggregated/sql.py

```diff
@@ -64,6 +64,16 @@ def DISTRIBUTED_TABLE_TEMPLATE(dist_table_name, base_table_name, columns):
 WEB_OVERVIEW_ORDER_BY = "(team_id, day_bucket, host, device_type)"
 WEB_STATS_ORDER_BY = "(team_id, day_bucket, host, device_type, os, browser, viewport, referring_domain, utm_source, utm_campaign, utm_medium, country)"
 
+WEB_BOUNCES_COLUMNS = """
```

Comment:
I wonder if this kind of constant implies we need some kind of parser in our tests to guarantee they exist